src/ipc/Kids.h

```diff
@@ -64,7 +64,7 @@ class Kids
 
 extern Kids TheKids; ///< All kids being maintained
 
-typedef char KidName[64]; ///< Squid process name (e.g., "squid-coord")
+typedef SBuf KidName; ///< Squid process name (e.g., "squid-coord")
```

Comment:
Do we have to typedef at all here? I do not see any other code making use of the type and callers need to know the type API to use it anyway. So it is not simplifying or making anything easier.

Reply:
Done.